### PR TITLE
update node instructions

### PR DESCRIPTION
**Pull request by finp**

### CODE REVIEW

**Comment by finp** [comment]
@evanshortiss  let's discuss - maybe better to separate pre-reqs for walkthru vs pre-reqs for those hacking on the walkthrough?

**Comment by evanshortiss** [comment]
@finp yeah I think so, though this is valid for the setup as a lab instructor. Maybe take a look [here](https://github.com/evanshortiss/rhte-2019-hackathon-on-rhmi/tree/master/content) and make the changes in that file?

**Comment by finp** [comment]
@evanshortiss  The reason I put it here is that the later in this readme you're asking user to npm install...
Actually, instead of moving it, I've added yarn ;)

File: README.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ This repository contains a Solution Pattern that's compatible with the
...
+## Requirements for development
+
+* Node.js 10+
+* npm 6+
+* yarn (see https://yarnpkg.com/en/docs/install)
+
+The easiest way to install these requirements is to use [nvm](https://github.com/nvm-sh/nvm#installation-and-update) and then
+
+```bash
+nvm use 10
```
Comment (evanshortiss) [inline README.md]: @finp thanks for catching the yarn/npm stuff. Could you add the yarn install like so?

```
nvm use 10
npm install -g yarn
```

Comment (finp) [reply README.md]: done

**Pull request merged**